### DISCUSSION TITLE
Import beatmaps/skins/replays from downloads folder on android

### DISCUSSION THREAD

**Discussion by kaanyalova** [ppy/osu#18091] [Ideas]
It is quite annoying to import beatmaps/skins/replays on android. It would be nice if osu scans the downloads folder for skin/beatmap/replay files on launch , similar to how osu droid does it.

**Comment by peppy** [comment]
You can already use file association or use the import button in settings.

But more importantly: is there a reason you aren't using the in-game beatmap listing rather than downloading beatmaps externally?

**Reply by kaanyalova** [reply]
I normally use in-game beatmap listing for beatmaps, but i think it would be easier for importing multiple skins.

**Reply by kaanyalova** [reply]
It would also make it easier to import replays.
